akka-actor/src/main/scala/akka/event/Logging.scala

```diff
@@ -1943,8 +1943,8 @@ class MarkerLoggingAdapter(
 
   // Copy of LoggingAdapter.format1 due to binary compatibility restrictions
   private def format1(t: String, arg: Any): String = arg match {
-    case a: Array[_] if !a.getClass.getComponentType.isPrimitive => format(t, a.toIndexedSeq)
-    case a: Array[_]                                             => format(t, a.map(_.asInstanceOf[AnyRef]).toIndexedSeq)
+    case a: Array[_] if !a.getClass.getComponentType.isPrimitive => formatImpl(t, a.toSeq)
+    case a: Array[_]                                             => formatImpl(t, a.map(_.asInstanceOf[AnyRef]).toSeq)
```

Comment:
`toSeq` and `toIndexedSeq` should both lead to a wrapping `ArraySeq`, isn't the `formatImpl` change enough to fix the issue?

Reply:
I agree, yet as indicated it should be a copy of LoggingAdapter.format1:
`// Copy of LoggingAdapter.format1 due to binary compatibility restrictions`
But indeed, no effect so keep or change for me does not matter.

Reply:
Ah, I see and that is toSeq, ok fine!